plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -200,6 +200,18 @@ private function get_feature_definitions() {
 					'is_experimental' => false,
 					'disable_ui'      => true,
 				),
+				'rate_limit_checkout'    => array(
+					'name'               => __( 'Rate limit Checkout', 'woocommerce' ),
+					'description'        => sprintf(
+						// translators: %s is the URL to the rate limiting documentation.
+						__( 'Enables rate limiting for Checkout place order and Store API /checkout endpoint. To further control this, refer to <a href="%s" target="_blank">rate limiting documentation</a>.', 'woocommerce' ),
+						'https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/src/StoreApi/docs/rate-limiting.md'
+					),
+					'is_experimental'    => false,
```

Comment:
Suggestion: Since this change seems unfinished, probably overriding config previously set and given the risk associated with enabling regarding proxies, should we move it to the Experimental section?